console/console-init/ui/src/modules/endpoints/EndpointPage.tsx

```diff
@@ -68,7 +69,10 @@ export default function AddressPage() {
     }
   };
   return (
-    <PageSection variant={PageSectionVariants.light}>
+    <PageSection
```

Comment:
Page section doesn't need id

Reply:
It's not fixed.

Reply:
Will be removing page section id in any future file chages